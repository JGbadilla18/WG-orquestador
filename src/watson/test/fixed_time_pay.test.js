const testWatson = require('./testWatson');

watsonText = 'Plazo fijo'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});

watsonText2 = 'Plazo fijo dolar'

it(watsonText2, () => {
  expect.assertions(1);
  return testWatson(watsonText2).then(data => expect(data).toMatchSnapshot());
});

watsonText3 = 'Plazo fijo quetzales'

it(watsonText3, () => {
  expect.assertions(1);
  return testWatson(watsonText3).then(data => expect(data).toMatchSnapshot());
});