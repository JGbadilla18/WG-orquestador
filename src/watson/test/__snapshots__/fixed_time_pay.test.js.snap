// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plazo fijo 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para solicitar un Plazo Fijo debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:
  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  Monto de apertura según el plazo",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/flazo-fijo-en-quetzales",
            },
          ],
          "image_url": " http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-Q.jpg",
          "subtitle": " Inversión en quetzales a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/plazo-fijo-en-dolares",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-.jpg",
          "subtitle": "Inversión en dólares a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`Plazo fijo dolar 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para solicitar un Plazo Fijo debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:
  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  Monto de apertura según el plazo",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/plazo-fijo-en-dolares",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-.jpg",
          "subtitle": "Inversión en dólares a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Dólares",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/flazo-fijo-en-quetzales",
            },
          ],
          "image_url": " http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-Q.jpg",
          "subtitle": " Inversión en quetzales a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Quetzales",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`Plazo fijo quetzales 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para solicitar un Plazo Fijo debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:
  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  Monto de apertura según el plazo",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/flazo-fijo-en-quetzales",
            },
          ],
          "image_url": " http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-Q.jpg",
          "subtitle": " Inversión en quetzales a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/plazo-fijo-en-dolares",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Plazo-fijo-.jpg",
          "subtitle": "Inversión en dólares a través de un Certificado de Depósito a plazo.",
          "title": "Plazo Fijo Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
